src/Model.php

```diff
@@ -15,31 +18,49 @@ class Model
 {
     protected array $data = [];
 
-    protected array $modified = [];
+    protected array $casts = [];
```

Comment:
I think `$dataTypes` would be a better name for `$casts`.

Reply:
More then happen to change it, only reason I called it `$casts` is because I mainly work with Laravel and that's what it's called there.